flow/Net2.actor.cpp

```diff
@@ -135,6 +135,12 @@ thread_local INetwork* thread_network = 0;
 
 class Net2 final : public INetwork, public INetworkConnections {
 
+private:
+	void updateStarvationTrackers(struct NetworkMetrics::PriorityStats& binStats,
```

Comment:
Minor nit -- I think we're updating a single tracker here

```suggestion
	void updateStarvationTracker(struct NetworkMetrics::PriorityStats& binStats,
```